packages/siop-oid4vp/lib/request-object/Payload.ts

```diff
@@ -49,6 +49,7 @@ export const createRequestObjectPayload = async (opts: CreateAuthorizationReques
     claims,
     presentation_definition_uri: payload.presentation_definition_uri,
     presentation_definition: payload.presentation_definition,
+    dcql_query: payload.dcql_query,
```

Comment:
This assumes a dcql_query is always present, returning in an undefined, whilst this whole function is about removing any nulls/undefineds

Reply:
I think you got me wrong. From the project management perspective it makes more sense to have the person who wrote the code to fix its issues because it's more efficient. (At the point I've wrote the comment above, there was an open PR and it was even reviewed). I should have only started to work on this code after the original PR was merged to avoid dependency on tickets, to reduce complexity and conflicts. It's addressed